src/test/ui/resolve/use_suggestion_placement.rs

```diff
@@ -7,7 +10,11 @@ mod m {
 }
 
 mod foo {
-    #[derive(Debug)]
+    // FIXME: UsePlacementFinder is broken because active attributes are
+    // removed, and thus the `derive` attribute here is not in the AST.
+    // An inert attribute should work, though.
+    // #[derive(Debug)]
```

Comment:
What does this look like uncommented in the fixed output?

Reply:
It looks like this (after formatting):

```rust
    #[derive(Debug)]
    use std::path::Path;
    #[allow(warnings)]
    pub struct Foo;
```